src/solvers/spqr.jl

```diff
@@ -132,7 +132,7 @@ struct QRSparseQ{Tv<:CHOLMOD.VTypes,Ti<:Integer} <: LinearAlgebra.AbstractQ{Tv}
     τ::Vector{Tv}
     n::Int # Number of columns in original matrix
 end
-
+Base.print_array(io::IO, Q::QRSparseQ) = Base.print_array(io, sparse(Q))
```

Comment:
I'm not sure we need this anymore. Neither `show` of `QRSParse` nor of `QRSparseQ` displays any kind of matrix representation for `Q`. Are there other paths that lead to this where this could be useful?

Reply:
The more I think about it, the more I believe this should be removed. Who wants to see a matrix representation can simply call `sparse(Q)` and that's it.